vars/warnAboutDeprecatedChartConfig.groovy

```diff
@@ -35,4 +39,15 @@ def call(Map<String, String> params) {
   }
   sh 'rm -f check-deprecated-charts.sh'
 
+  sh 'chmod +x check-deprecated-gradle-dependency.sh'
+  gradleDeprecationConfig.each { dependency, deprecatedVersions ->
+    deprecatedVersions.each { deprecation ->
+      try {
+        sh "./check-deprecated-gradle-dependency.sh $dependency $deprecation.version "
+      } catch (ignored) {
+        WarningCollector.addPipelineWarning("deprecated_helmcharts", "Versions below $dependency: $deprecation.version are deprecated, please upgrade to latest release https://github.com/hmcts/$dependency/releases", LocalDate.parse(deprecation.date_deadline))
```

Comment:
I don't know if you can assume it will be an HMCTS repo?
```suggestion
        WarningCollector.addPipelineWarning("deprecated_gradle_library", "Versions below $dependency: $deprecation.version are deprecated, please upgrade to the latest release", LocalDate.parse(deprecation.date_deadline))
```

Reply:
Good point, this line is specific to dependencies we'd manage - thanks :)